### ISSUE TITLE
Missing libresolv.so Reference with fips enabled

### ISSUE THREAD

**Issue by KrisJohnstone** [projectcalico/calico#8515]
It would appear that the node-driver-registrar container is missing a reference to libresolv.so

## Expected Behavior
Expect the container to start without any error (or at least not one related to libresolv).

## Current Behavior
```
/usr/local/bin/node-driver-registrar: error while loading shared libraries: libresolv.so.2: cannot open shared object file: No such file or directory
```

## Possible Solution
From what I can see this might be resolved by copying libresolv into the container like it does for libc.so, which means by adding
```
COPY --from=ubi /lib64/libresolv.so.2 /lib64/libresolv.so.2
```

The problem might be resolved. Unfortunately I can't get it to build on my mac so can't test it.

## Steps to Reproduce (for bugs)

Set fipsMode to enabled in the installation:
```
  installation:
    fipsMode: Enabled
```

## Your Environment
* Calico version - 3.27.0
* Orchestrator version - Kubernetes (EKS)
* Operating System and version: 1.27.0

**Comment by hjiawei** [comment]
Thank you for reporting this @KrisJohnstone. We have created https://github.com/projectcalico/calico/pull/8516 to address this.

**Comment by mazdakn** [comment]
Closing the issue, since the fix is merged now. It will be released in calico v3.27.2 patch release which we expect to happen soon.
